lib/robots/dor_repo/gis_assembly/generate_content_metadata.rb

```diff
@@ -55,8 +55,19 @@ def epsg_data_zip_objectfile
           @epsg_data_zip_objectfile ||= Assembly::ObjectFile.new("#{rootdir}/content/data_EPSG_4326.zip")
         end
 
+        def preview_objectfile_path
+          thumbnail_file = File.join(rootdir, 'content', 'preview.jpg')
+          return thumbnail_file if File.size?(thumbnail_file)
+
+          temp_thumbnail_file = File.join(rootdir, 'temp', 'preview.jpg')
+          raise "generate_content_metadata: #{bare_druid} is missing thumbnail preview.jpg" unless File.size?(temp_thumbnail_file)
+
+          FileUtils.cp(temp_thumbnail_file, thumbnail_file)
+          thumbnail_file
```

Comment:
code climate is saying that these newly added (or just moved from `ExtractThumbnail`?) lines are the uncovered ones.